packages/auth/migrations/20220920224349_add_resource_id_to_accesses_table.js

```diff
@@ -0,0 +1,21 @@
+exports.up = function (knex) {
+  return knex.schema.alterTable('accesses', function (table) {
```

Comment:
Was initially wary of messing with the dates on the migration files since `create_accesses_table` would have to come after `create_resource_sets_table`, but moving them around like that doesn't seem to be an issue.